.github/workflows/ci.yml

```
@@ -13,15 +13,19 @@ on:
 
 jobs:
   pest:
-    name: Tests (Pest)
+    name: Tests (Pest) L${{ matrix.laravel }}
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        laravel: [9, 8]
```

We'll use ascending order. I'll also fix this in the helpers & template repos
```suggestion
        laravel: [8, 9]
```

---

Thanks, I'll follow this from now on.